app/views/shared/_claim_history.html.haml

```diff
@@ -6,8 +6,8 @@
   - message_controls_url = show_message_controls_external_users_claim_path(claim)
 
 .messages-container.no-print
-  .messages-list
-    = render partial: 'shared/history_items_list', locals: {claim: claim }
+  .messages-list{ 'aria-label': t('shared.claim_accordion.h2_messages'), role: 'region', tabindex: 0 }
```

Comment:
**not a blocker**. does the claim_accordion still exist. is this locales entry just an outdated name now?

Reply:
Yes, claim_accordion exists. Please note we have a ticket to refactor the locales file.